iroh-bitswap/src/client/provider_query_manager.rs

```diff
@@ -75,7 +75,7 @@ impl ProviderQueryManager {
         &self,
         cid: &Cid,
     ) -> Result<async_channel::Receiver<Result<PeerId>>> {
-        let (s, r) = async_channel::bounded(8);
+        let (s, r) = async_channel::bounded(10);
```

Comment:
Why does 10 help more, should it be configurable or at least extracted as a const?

Reply:
I will make it a constant eventually, it's just to match the number of providers we are fetching, they now fit all into the channel.